src/docfx/build/Build.cs

```diff
@@ -22,11 +22,18 @@ public static async Task<int> Run(string workingDirectory, CommandLineOptions op
                 return 1;
             }
 
-            var result = await Task.WhenAll(docsets.Select(docset => BuildDocset(docset.docsetPath, docset.outputPath, options)));
-            return result.All(x => x) ? 0 : 1;
+            var hasError = false;
+            Parallel.ForEach(docsets, docset =>
+            {
+                if (BuildDocset(docset.docsetPath, docset.outputPath, options))
+                {
+                    hasError = true;
```

Comment:
Since this is shared between threads, would there be race condition?
But we only care about the `true` case, overwrite would not happen, it should be fine...